include/rocksdb/statistics.h

```diff
@@ -388,6 +388,11 @@ enum Tickers : uint32_t {
   MEMTABLE_PAYLOAD_BYTES_AT_FLUSH,
   // Outdated bytes of data present on memtable at flush time.
   MEMTABLE_GARBAGE_BYTES_AT_FLUSH,
+  // Counter for how many times the MemPurge operation is performed.
```

Comment:
If users cannot use this its better to add experimental otherwise it's fine.

Reply:
Great idea - done

Reply:
Except this is among the most public of public APIs. Because of lots of downstream infrastructure built on these statistics, AFAIK additions are quite permanent. I'm not sure of all the consequences of changing a name.